stubs/docker/docker/types/daemon.pyi

```diff
@@ -1,6 +1,11 @@
-class CancellableStream:
+from collections.abc import Iterator
+from typing import Generic, TypeVar
+
+_T = TypeVar("_T")
```

Comment:
I think this typevar should be covariant:

```suggestion
_T_co = TypeVar("_T_co", covariant=True)
```

Reply:
Thanks for teaching me this!